updates/1.141/230929-communicator-trash-timer.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Seems OK but version 1.141 has already been claimed.

Reply:
Should be fine now